app/models/message.rb

```diff
@@ -0,0 +1,6 @@
+class Message < ActiveRecord::Base
+  belongs_to :conversation
+  belongs_to :user
+
+  validates_presence_of :body, :conversation_id, :user_id
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.